### PR TITLE
Small refactoring / cleanup

### PR DESCRIPTION
**Pull request by winder**
## Summary

Some safe and relatively uncontroversial refactoring. Relates to #279 

## Test Plan

No new functionality to test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,28 +3,28 @@ go:
 - 1.13.x
```
Comment (brianolson) [inline .travis.yml:3]: we should definitely drop 1.13 now. 1.16 is out and only 1.16 and 1.15 will be getting fixes now.

File: idb/postgres/postgres.go [hunk 2]
```diff
@@ -375,8 +375,8 @@ func (db *IndexerDb) GetProto(version string) (proto types.ConsensusParams, err
 	return
 }
 
-// GetMetastate is part of idb.IndexerDB
-func (db *IndexerDb) GetMetastate(key string) (jsonStrValue string, err error) {
+// getMetastate is part of idb.IndexerDB
```
Comment (brianolson) [inline idb/postgres/postgres.go:378]: drop this comment line

File: idb/postgres/postgres.go [hunk 3]
```diff
@@ -390,12 +390,37 @@ func (db *IndexerDb) GetMetastate(key string) (jsonStrValue string, err error) {
 
 const setMetastateUpsert = `INSERT INTO metastate (k, v) VALUES ($1, $2) ON CONFLICT (k) DO UPDATE SET v = EXCLUDED.v`
 
-// SetMetastate is part of idb.IndexerDB
-func (db *IndexerDb) SetMetastate(key, jsonStrValue string) (err error) {
+// setMetastate is part of idb.IndexerDB
```
Comment (brianolson) [inline idb/postgres/postgres.go:393]: also not quite true anymore

**Review by brianolson — COMMENTED** [review]
lots of good stuf. a couple tweaks

File: .travis.yml (hunk 1 above)
Comment (winder) [reply .travis.yml]: Good idea. Dropped 1.13, looks like there's some minor work to get 1.16 working so I'll leave that off for now.

**Review by brianolson — APPROVED** [review]

**Pull request merged**